model/status_board/v1/service_info_type.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Definition of a Status Board service info.
+class ServiceInfo {
+  // Full name of the service
+  Fullname string
+
+  // Type of the service status
+  StatusType string
+}
```

Comment:
Using a `class` here instead of a `struct` means that this is anentity has its own identity, and in practical terms that it will have the `id` and `href` attributes. With my limited understanding of your service I'd say that is not what you want or need. So better replace `class` with `struct`.

Reply:
Thank you, `class` replaced with `struct`.